docs/integration/solr-logs.md

```diff
@@ -69,9 +31,3 @@ Other built-in reports include:
 - **Zookeeper**: logs produced by Solr's [Zookeeper](https://zookeeper.apache.org) client and other Zookeeper-related classes and threads. Look here for insights on SolrCloud's stability. For the Zookeeper ensemble itself, check out our [Zookeeper monitoring integration](./zookeeper.md)
 - **Overseer**: logs produced by SolrCloud's [Overseer](https://lucene.apache.org/solr/8_6_0/solr-core/org/apache/solr/cloud/Overseer.html) thread and other Overseer activities (e.g. shard leader election)
 - **Start & Stop**: startup-related and shutdown-related logs. Look here if a node went down unexpectedly or doesn't show up in the cluster when started
-
-## Troubleshooting
```

Comment:
Wonder if it would be wise to keep a Troubleshooting section (and make sure we have it on on relevant integration pages) and include a link to the Fleet page under /docs?

Reply:
Updated, we will keep/add a troubleshooting section in all integrations.